onnxruntime/python/tools/transformers/models/llama/benchmark_all.py

```diff
@@ -211,14 +223,52 @@ def save_results(results, filename):
     )
 
     # Set column types
+    df["Warmup Runs"] = df["Warmup Runs"].astype("int")
+    df["Measured Runs"] = df["Measured Runs"].astype("int")
     df["Batch Size"] = df["Batch Size"].astype("int")
     df["Sequence Length"] = df["Sequence Length"].astype("int")
     df["Latency (s)"] = df["Latency (s)"].astype("float")
     df["Latency (ms)"] = df["Latency (ms)"].astype("float")
     df["Throughput (tps)"] = df["Throughput (tps)"].astype("float")
     df["Memory (GB)"] = df["Memory (GB)"].astype("float")
 
-    df.to_csv(filename, index=False)
+    # get pakcage name and version
+    import pkg_resources
```

Comment:
For any new libraries that are imported such as `pkg_resources`, can you check that they can be imported after installing the package list in the `requirements.txt` file with `pip install -r requirements.txt`? If not, we need to add any missing packages there.

Reply:
the package is 'setuptools', it is usually installed by python by default.